packages/core/src/components/tabs/tabs.tsx

```diff
@@ -0,0 +1,157 @@
+import { Component, Element, Event, EventEmitter, Fragment, h, Method, State } from "@stencil/core";
+import clsx from "clsx";
+
+import { Tab, TabsSwitchEvent } from "./tabs.interfaces";
+
+@Component({
+  tag: "dso-tabs",
+  styleUrl: "tabs.scss",
+  shadow: true,
+})
+export class Tabs {
+  private tabs?: Tab[];
+
+  @Element()
+  host!: HTMLDsoTabsElement;
+
+  /**
+   * Emitted when the user activates tab via click or arrow keys.
+   */
+  @Event()
+  dsoTabSwitch!: EventEmitter<TabsSwitchEvent>;
+
+  @State()
+  activeTab?: string;
```

Comment:
Waar gebruiken we deze state property voor?

Reply:
Nergens voor. Heb ik weg gehaald